cancer/rank_model/cancer_rank_model_-v1.2-.ini

```
@@ -0,0 +1,454 @@
+    score = 1
+    priority = 2
+    string = 'regulatory_region_variant'
+
+  [[upstream_gene_variant]]
+    score = 1
+    priority = 2
+    string = 'upstream_gene_variant'
+
+  [[regulatory_region_amplification]]
+    score = 1
+    priority = 2
+    string = 'regulatory_region_amplification'
+
+  [[tfbs_amplification]]
+    score = 1
+    priority = 2
+    string = 'tfbs_amplification'
+
+  [[5_prime_utr_variant]]
+    score = 1
+    priority = 2
+    string = '5_prime_utr_variant'
+
+  [[intron_variant]]
+    score = 1
+    priority = 2
+    string = 'intron_variant'
+
+  [[3_prime_utr_variant]]
+    score = 1
+    priority = 2
+    string = '3_prime_utr_variant'
+
+  [[feature_truncation]]
+    score = 1
+    priority = 2
+    string = 'feature_truncation'
+
+  [[TF_binding_site_variant]]
+    score = 1
+    priority = 2
+    string = 'TF_binding_site_variant'
+
+  [[stop_retained_variant]]
+    score = 1
+    priority = 2
+    string = 'stop_retained_variant'
+
+  [[feature_elongation]]
+    score = 1
+    priority = 2
+    string = 'feature_elongation'
+
+  [[regulatory_region_ablation]]
+    score = 1
+    priority = 2
+    string = 'regulatory_region_ablation'
+
+  [[tfbs_ablation]]
+    score = 1
+    priority = 2
+    string = 'tfbs_ablation'
+
+  [[coding_sequence_variant]]
+    score = 1
+    priority = 2
+    string = 'coding_sequence_variant'
+
+  [[downstream_gene_variant]]
+    score = 1
+    priority = 2
+    string = 'downstream_gene_variant'
+
+  [[NMD_transcript_variant]]
+    score = 1
+    priority = 2
+    string = 'NMD_transcript_variant'
+
+  [[intergenic_variant]]
+    score = 0
+    priority = 0
+    string = 'intergenic_variant'
+
+  [[not_reported]]
+    score = 0
+
+[filter]
+  category = variant_call_quality_filter
+  data_type = string
+  description = The filters for the variant
+  field = FILTER
+  record_rule = min
+  separators = ';',
+
+  [[not_reported]]
+    score = 0
+
+  [[pass]]
+    score = 3
+    priority = 1
+    string = 'PASS'
+
+[coverage]
+  category = variant_call_quality_filter
+  data_type = integer
+  description = Coverage on position
+  field = INFO
+  info_key = DP
+  record_rule = min
+  separators = ',',
+
+  [[not_reported]]
+    score = 0
+
+  [[low]]
+    score = -6
+    priority = 1
+    upper = 100
+    lower = -1
+
+  [[high]]
+    score = 0
+    priority = 1
+    upper = 10000000
+    lower = 100
+
+[mapping_quality]
+  category = variant_call_quality_filter
+  data_type = integer
+  description = Mapping quality
+  field = INFO
+  info_key = MQ
+  record_rule = min
+  separators = None
+
+  [[not_reported]]
+    score = 0
+
+  [[ok]]
+    score = 0
+    lower = 60
+    upper = 1000000
+
+  [[intermediate]]
+    score = -2
+    lower = 40
+    upper = 59
+
+  [[poor]]
+    score = -4
+    lower = -1
+    upper = 40
+
+[variant_allele_frequency]
+  category = lineage
+  data_type = float
+  description = Variant allele frequency
+  field = INFO
+  info_key = ADJAF
```

If this is for VarDict, I suggest using AF

```suggestion
  info_key = AF
```

---

I trust you there; it looked like the more advanced one, but if you have experience otherwise! VarDict was the only one that was in the Scout load config, so I assumed that was the one in use?